### PR TITLE
Add flags to enable/disable the build of ck-benchmark-driver and rocblas-benchmark-driver

### PR DESCRIPTION
**Pull request by yiqian1**
The flags MLIR_USE_CK and MLIR_USE_ROCBLAS to disable building  ck-benchmark-driver and rocblas-benchmark-driver by default. Enable them in Jenkinsfile.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -35,6 +35,8 @@ set(ROCMLIR_BIN_DIR "${CMAKE_CURRENT_BINARY_DIR}/bin" CACHE PATH "")
 set(ROCMLIR_LIB_DIR "${CMAKE_CURRENT_BINARY_DIR}/lib" CACHE PATH "")
 message(STATUS "ROCMLIR_BIN_DIR: ${ROCMLIR_BIN_DIR}")
 
+set(MLIR_USE_CK OFF CACHE BOOL "Enable build CK")
```
Comment (krzysz00) [inline CMakeLists.txt]: I don't quite like these names. Can we go for `ROCMLIR_ENABLE_BENCHMARKS` and then samething like `ROCMLIR_ENABLE_CK_BENCHMARKS`/`ROCMLIR_ENABLE_ROCBLAS_BENCHMARK` 

Or, alternatively, `ROCMLIR_ENABLE_BENCHMARKS=all` or `=ck;rocblas` or `=rocblas` or `=ck` or `=""` (to disable)

File: mlir/utils/performance/ck-benchmark-driver/CMakeLists.txt [hunk 2]
```diff
@@ -1,17 +1,19 @@
-find_package(composable_kernel CONFIG 1.0.0 COMPONENTS device_operations)
-if(NOT composable_kernel_FOUND)
-  message(WARNING "Could not find composable-kernels, not building a CK benchmark driver")
-endif()
+if(MLIR_USE_CK)
```
Comment (krzysz00) [inline mlir/utils/performance/ck-benchmark-driver/CMakeLists.txt]: I'd like to move this `if` one level up, into the `CMakeLists.txt` for `performance/`

**Review by krzysz00 — COMMENTED** [review]
I've got naming complaints

File: CMakeLists.txt (hunk 1 above)
Comment (yiqian1) [reply CMakeLists.txt]: I agree. I've made the changes and pushed them. But somehow this PR is not updated.

Comment (yiqian1) [reply CMakeLists.txt]: Finally it got  updated.

File: CMakeLists.txt [hunk 3]
```diff
@@ -30,11 +30,17 @@ set(ROCMLIR_DRIVER_RANDOM_DATA_SEED "none" CACHE STRING "Enable E2E tests using
...
 set(ROCMLIR_DRIVER_TEST_GPU_VALIDATION 1 CACHE BOOL "Enable E2E tests with GPU validation")
 set(ROCK_E2E_TEST_ENABLED 0 CACHE BOOL "Enable build rock E2E tests")
+set(ROCMLIR_ENABLE_BENCHMARKS "" CACHE STRING "List of enabled benchmarks")
 
 set(ROCMLIR_BIN_DIR "${CMAKE_CURRENT_BINARY_DIR}/bin" CACHE PATH "")
 set(ROCMLIR_LIB_DIR "${CMAKE_CURRENT_BINARY_DIR}/lib" CACHE PATH "")
 message(STATUS "ROCMLIR_BIN_DIR: ${ROCMLIR_BIN_DIR}")
 
+if( ROCMLIR_ENABLE_BENCHMARKS STREQUAL "all" )
+  message(STATUS "set benchmarsk")
```
Comment (krzysz00) [inline CMakeLists.txt]: We might want a more helpful message like "enabling all benchmarks" or none at all.

**Review by krzysz00 — APPROVED** [review]
Looks good to me, approved with one nit

**Comment by sjw36** [comment]
Thanks @yiqian1!

**Pull request merged**